inky/types.go

```diff
@@ -1,9 +1,9 @@
 // Copyright 2023 The Periph Authors. All rights reserved.
 // Use of this source code is governed under the Apache License, Version 2.0
 // that can be found in the LICENSE file.
+package inky
```

Comment:
I meant to still include an empty line above, otherwise this will be pulled as documentation when you look at https://pkg.go.dev/periph.io/x/devices/v3/inky.

Reply:
Grrr - I thought I did that already like the rest of the files, fixed :)